### PR TITLE
Edited documentation for pointer events

### PR DESCRIPTION
**Pull request by LadySith**
Edited documentation for touchStarted(), touchPressed(), mouseMoved()
and mouseDragged() to specify inclusion of return true to enable default
behaviors when using methods.

### CODE REVIEW

File: src/events/mouse.js [hunk 1]
```diff
@@ -466,6 +466,8 @@ p5.prototype._setMouseButton = function(e) {
  * Browsers may have different default
  * behaviors attached to various mouse events. To prevent any default
  * behavior for this event, add "return false" to the end of the method.
+ * Otherwise add "return true" if you wish for default
+ * mouse behavior to be recognized by browser.
```
Comment (lmccart) [inline src/events/mouse.js]: I thought that the default behavior would occur if there was no return line. Or do I have that wrong?

Comment (limzykenneth) [reply src/events/mouse.js]: Returning nothing should give the default behavior as the code only check for false with `===` so any other value (including `undefined` when there is no return) should give default behaviour

**Comment by limzykenneth** [comment]
Hi @LadySith, can you explain a bit about this PR? The events functions doesn't need the user to return true to enable default behavior, only return false to disable them.

**Comment by LadySith** [comment]
Hi @limzykenneth and @lmccart . Sorry I was trying to find a solution to this [Issue 2751](https://github.com/processing/p5.js/issues/2751) which turned out to be a non-issue. Since the default is to enable the behavior then this pull request can be closed.

**Pull request closed without merging**